SoftLayer/CLI/hardware/edit.py

```diff
@@ -62,3 +70,9 @@ def cli(env, identifier, domain, userfile, tag, hostname, userdata,
 
     if private_speed is not None:
         mgr.change_port_speed(hw_id, False, int(private_speed))
+
+    if private_trunk is not None:
+        mgr.trunk(hw_id, False, int(private_trunk))
+
+    if public_trunk is not None:
+        mgr.trunk(hw_id, True, int(public_trunk))
```

Comment:
I think this int() casting is not needing since the type the click library already validates for an int.